bin/lowly.js

```diff
@@ -5,8 +5,12 @@ const path = require('path')
 const filesize = require('filesize')
 const meow = require('meow')
 const pify = require('pify')
+const mmm = require('mmmagic')
```

Comment:
Maybe you could change the library, because `mmmagic` rely on [node-gyp](https://github.com/nodejs/node-gyp) for compile the native addons used. That can be a problem when you are running [windows](https://github.com/nodejs/node-gyp/issues/154) beacause you must have installed properly both the `node-gyp` and the [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) for node. So, I think that `mmmagic` adds unnecessary complexity.

An library that does almost the same thing is [file-type](https://www.npmjs.com/package/file-type) but doesn't rely on native addons.